aviary/validation_cases/benchmark_tests/test_bench_FwFm.py

```diff
@@ -364,6 +374,20 @@ def test_bench_FwFm_SNOPT(self):
         compare_against_expected_values(prob, self.expected_dict)
 
 
+@use_tempdirs
+@unittest.skipUnless(MPI and PETScVector, "MPI and PETSc are required.")
+class TestBenchFwFmParallel(ProblemPhaseTestCase):
+
+    N_PROCS = 3
```

Comment:
just curious, but why 3?

Reply:
Dymos' "phases" is a parallel group, so this is 1 proc for each phase.